sysml.library/Kernel Libraries/Kernel Semantic Library/Occurrences.kerml

```diff
@@ -1,16 +1,16 @@
-package Occurrences {
```

Comment:
Something weird happened to this file, but the content seems to be the same.

Reply:
@himi noticed on a different branch that `Occurrences.kerml` had inconsistent line terminators (this happens every so often because of the barbarians using Windows! 😉). He fixed it on the other branch, and I cherry-picked that commit into this branch, to make sure his changes were preserved along with the additional update to a "standard library package".